scripts/common/rook.sh

```diff
@@ -216,7 +234,10 @@ function maybe_cleanup_rook() {
 
         if [ "$DID_MIGRATE_ROOK_PVCS" == "1" ] && [ "$DID_MIGRATE_ROOK_OBJECT_STORE" == "1" ]; then
             report_addon_start "rook-ceph-removal" "v1"
-            remove_rook_ceph
+            if ! remove_rook_ceph; then
+                logFail "Unable to remove Rook."
+                return
```

Comment:
this would fail before but now you are handling this error. is that intentional? do you mean to return 1 here?

Reply:
The migrations were done with success and we face an issue to delete/remove Rook
Why we should fail the whole upgrade because we were unable to remove Rook? 
It is possible to fix that manually afterwords.

Reply:
That would not report success because we would return prior 242.
But I can do another PR only for that for sure.